src/main/java/Task.java

```diff
@@ -0,0 +1,37 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public boolean isDone() {
+        return isDone;
+    }
+
+    public void setDone(boolean done) {
+        isDone = done;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " ");
+    }
+
+    public void markAsDone() {
+        isDone = true;
+    }
+
+    public void unmarkAsDone() {
```

Comment:
```suggestion
    public void markAsUndone() {
```
Prefer intention-revealing methods for state changes. For naming, use verbs for actions.